functional_testing/Openvcloud/utils/utils.py

```diff
@@ -524,6 +524,17 @@ def get_running_stackId(self, except_stackid=''):
                 return stackId
         return False
 
+    def get_random_running_nodeId(self):
```

Comment:
u can use "get_nodeId_to_move_VFW_to" instead of duplicating .. if u don't like the naming, u can just change it every where

Reply:
It takes "current_VFW_nodeId" parameter that is why I didn't use it?.
Maybe we can set an initialization value to it. What do u think?

Reply:
I think it will be better to initialize current_VFW_nodeId and make it generic